src/views/Table.vue

```diff
@@ -0,0 +1,187 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <div>
+    <CylcObjectMenu />
```

Comment:
This is the mutations menu. As it's already in the Table view, we should — in theory — have the menu working with the directive :crossed_fingers: 

Will send the diff later after some of these comments are reviewed/resolved.

Reply:
Mutations working now.